wifi-connect/Dockerfile.template

```diff
@@ -0,0 +1,37 @@
+FROM balenalib/%%BALENA_ARCH%%-python as builder
+
+RUN install_packages \
+    dnsmasq \
+    wireless-tools \
+    git \
+    rsync \
+    binutils \
+    busybox-static\
+    wget \
+    libzmq3-dev
+
+WORKDIR /usr/src/app
+
+RUN curl https://api.github.com/repos/balena-io/wifi-connect/releases/latest -sL \
+    | grep -hoP 'browser_download_url": "\K.*%%BALENA_ARCH%%\.tar\.gz' \
+    | xargs -n1 curl -Ls \
+    | tar -xvz -C /usr/src/app/
+
+COPY requirements.txt /tmp/requirements.txt
+RUN pip install --no-cache-dir -r /tmp/requirements.txt
+
+COPY trigger_wifi_setup.py .
+
+RUN pip install git+https://github.com/larsks/dockerize
```

Comment:
Avoid using anything that uses GPL3, as it creates a lot of spill-over issues.

Reply:
Got it. I was planning on using [balena-os/wifi-connect][1], but it uses _Apache-2.0_, [which is not compatible with _GPL-2.0_][2] (the one that _Anthias_ uses).

Looks like a need to rework on some changes.

[1]: https://github.com/balena-os/wifi-connect
[2]: https://www.gnu.org/licenses/license-list.en.html#apache2

Reply:
However, the current changes use [balena-labs-projects/wifi-connect][1], which doesn't explicitly include a license in its GitHub page, but uses [dockerize][2], which uses GPL-3.0 (as you've mentioned earlier). Hmmm. Will look again for alternatives.

[1]: https://github.com/balena-labs-projects/wifi-connect
[2]: https://github.com/larsks/dockerize